package.json

```diff
@@ -74,7 +74,7 @@
     "replace-in-file": "3.4.4",
     "request": "2.88.0",
     "request-promise-native": "1.0.7",
-    "restify": "8.4.0",
+    "restify": "^11.1.0",
```

Comment:
most of the packages are pinned to a specific version. i'd recommend to also keep that for this package.

Reply:
done